docs/docs/contribute/expect-during-pr.mdx

```diff
@@ -0,0 +1,37 @@
+# What to expect during a Pull request review
+
+At CLI for Microsoft 365, we appreciate your contributions through pull requests (PRs) for the issues you're helping us with. The review process is a collaborative effort between you and the maintainers who review your code. Depending on availability, your PR may be reviewed by one or more reviewers.
+
+We prioritize processing PRs in the order they are received (first in, first out). PRs with bug fixes are given priority over new features and enhancements because they address active issues that impact our users. When a PR requires adjustments after review, it is moved to the end of the processing queue to prevent blocking the review process.
+
+This article aims to provide an understanding of how we review pull requests, what you can expect during the review process, and how a pull request is merged or closed.
+
+## What happens immediately after submitting a PR?
+
+Once you submit a PR, one of our maintainers will examine it and check if the PR build has passed. The automated checks include:
+
+- Successful code build
+- Passing automated tests with 100% coverage
+- Absence of issues caught by our linter (ensuring adherence to naming conventions and code consistency)
+
+PRs are reviewed in the order of their submission, and once reviewed, they are moved to the end of the queue. If issues are identified during the review, the maintainers promptly notify the author to rectify them before proceeding with the review.
+
+## Review Process and Feedback
+
+When your PR is under review, the assigned maintainer will indicate their involvement by assigning themselves to the PR. They will follow a checklist for the review process, which can be found [here](https://github.com/pnp/cli-microsoft365/wiki/PR-checklist).
+
+Maintainers may also provide suggestions for best practices to ensure consistent coding styles. If there are code-related comments, you will see "changes requested" next to your username in the PR. In case the maintainer needs your input on an issue before continuing the review, the PR will be labeled as "waiting on response." The reviewer may also mark the pull request as **Draft** to indicate that it is a work in progress and changes are expected.
```

Comment:
```suggestion
Maintainers may also provide suggestions for best practices to ensure consistent coding styles. If there are code-related comments, you will see "changes requested" next to your username in the PR. In case the maintainer needs your input on an issue before continuing the review, the PR will be labelled as "waiting on response." The reviewer will also mark the pull request as **Draft** to indicate that it is `work in progress` and that changes are expected to be made.
```